src/Components/3DV/SceneView.tsx

```diff
@@ -81,35 +83,30 @@ function debounce(func: any, timeout = 300) {
 
 let dummyProgress = 0; // Progress doesn't work for GLBs so fake it
 
-const getModifiedTime = (url): Promise<string> => {
-    const promise = new Promise<string>((resolve) => {
-        const headers = new Headers();
-        headers.append('Range', 'bytes=1-2');
-        headers.append('x-ms-version', '2017-11-09');
-        if (Tools.CustomRequestHeaders.Authorization) {
-            headers.append(
-                'Authorization',
-                Tools.CustomRequestHeaders.Authorization
-            );
-        }
-
-        // HEAD can give a CORS error
-        fetch(url, { method: 'GET', headers: headers })
-            .then((response) => {
-                const dt = new Date(response.headers.get('Last-Modified'));
-                if (
-                    dt.toString() === 'Invalid Date' ||
-                    dt.toISOString() === '1970-01-01T00:00:00.000Z'
-                ) {
-                    resolve('');
-                }
-                resolve(dt.toISOString());
+const getModifiedTime = async (url): Promise<string> => {
+    return axios(url, {
+        method: 'GET',
+        headers: {
+            Range: 'bytes=1-2',
+            'x-ms-version': '2017-11-09',
+            ...(Tools.CustomRequestHeaders.Authorization && {
+                Authorization: Tools.CustomRequestHeaders.Authorization
             })
-            .catch(() => {
-                resolve('');
-            });
-    });
-    return promise;
+        }
+    })
+        .then((response) => {
+            const dt = new Date(response.headers['last-modified']);
+            if (
+                dt.toString() === 'Invalid Date' ||
+                dt.toISOString() === '1970-01-01T00:00:00.000Z'
+            ) {
+                return '';
+            }
+            return dt.toISOString();
+        })
+        .catch((error) => {
+            throw error;
```

Comment:
So far as I can tell, this looks a risky change as it promotes the fetch for the modified date to the front line, and fails the load if there's a network error fetching the modified date. I have seen MANY instances where this call fails, but the model load succeeds. I originally had something similar, but backed off that as it failed too many loads.

Please ignore this comment if that is not true.

Reply:
Also this line is technically wrong, but quite nice in practice.

mod = mod ? '?' + mod : '';

If we don't get modified date, we load without a query string which will never cache bust, so if you update the model, we will never load it if the previous version is in cache.

Now actually the getModifiedDate often fails and its a pain to have to keep reloading the model as most of the time, we haven't changed the model, so the behaviour is wrong, but IMHO desirable. If a customer meets this scenario, they can upload the new model with a new name and change their 3DScenesConfig (but better would be to fix their environment so the getModifiedDate works).

If we want to cache bust if the getModifiedDate fails, then this line could be

mod = mod ? '?' + mod : '?' + new Date().toIsoString();

but I don't like it. I've been meaning to discuss this on standup as I heard a rumour that one of our customers was hitting this.

Reply:
Looks like the line above `engine.disableManifestCheck = true;` is causing the BABYLON.SceneLoader.Load method not surfacing the network error in its onError callback. I asked it to Babylon people and waiting for their reply if they are interconnected. @ptallettms why do we need disabling manifest check, was it for caching related?

Reply:
Hey @ptallettms what is you final input on this mod related line? Is `mod = '?' + mod || new Date().toIsoString();` better option now?

Reply:
Whilst technically incorrect, I'd leave it as it is as that is what the user probably intended.

Reply:
thanks for your reply!